aibolit/patterns/var_siblings/var_siblings.py

```diff
@@ -1,26 +1,69 @@
-import javalang
-import re
-import itertools
-
+# The MIT License (MIT)
+#
+# Copyright (c) 2020 Aibolit
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included
+# in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+from aibolit.ast_framework import ASTNodeType, AST
 from aibolit.utils.ast_builder import build_ast
+from typing import List, Dict
+from aibolit.ast_framework.ast_node import ASTNode
+import re
 
 
 class VarSiblings:
-    def __init__(self):
-        pass
+    '''
+    Find those variables, which have complex
+    names and start with the same word
+    '''
+    def _collect_method_variables_names(self, ast: AST, node: ASTNode) -> Dict[str, int]:
+        assert node.node_type == ASTNodeType.METHOD_DECLARATION
+        vars_lines: Dict[str, int] = {}
+        for local_var_node in ast.get_proxy_nodes(ASTNodeType.LOCAL_VARIABLE_DECLARATION):
+            var_line = local_var_node.line
+            var_declaration = local_var_node.declarators[0]
```

Comment:
There might be declared several fields in single line. You could add them all to `var_lines`.

Reply:
done